testing/tests/DevExpress.ui.widgets.editors/dropDownBox.tests.js

```diff
@@ -493,21 +492,55 @@ QUnit.module('popup options', moduleConfig, () => {
 
         try {
             instance.open();
-
             this.clock.tick();
-            const popup = $('.dx-popup').dxPopup('instance');
+
+            const popup = this.$element.find('.dx-popup').dxPopup('instance');
             const maxHeight = popup.option('maxHeight');
+            assert.roughEqual(Math.floor(maxHeight()), startPopupHeight, 3, 'maxHeight is correct');
+        } finally {
+            scrollTop.restore();
+            this.$element.css('margin-top', 0);
```

Comment:
to my mind, it's not necessary because of [fixture updating before each test](https://github.com/DevExpress/DevExtreme/blob/21_1/testing/tests/DevExpress.ui.widgets.editors/dropDownBox.tests.js#L17)